lib/error.js

```diff
@@ -7,8 +7,6 @@ var messages = require('joi/lib/language').errors
 
 var ERRNO = {
   ACCOUNT_EXISTS: 101,
-  ACCOUNT_LOCKED: 121,
```

Comment:
@shane-tomlinson do we need to do the same here as content server and comment these out instead of deleting?